acme/src/test/groovy/io/micronaut/acme/AcmeCertRefresherTaskUnitSpec.groovy

```diff
@@ -7,12 +7,14 @@ import io.micronaut.runtime.event.ApplicationStartupEvent
 import io.micronaut.runtime.exceptions.ApplicationStartupException
 import io.netty.handler.ssl.util.SelfSignedCertificate
 import org.shredzone.acme4j.exception.AcmeException
+import spock.lang.Ignore
 import spock.lang.Specification
 import spock.lang.Stepwise
 import spock.lang.Unroll
 
 import java.time.Duration
 
+@Ignore
```

Comment:
I had to ignore this test. Can you investigate why it fails?

Reply:
I've re-enabled it